SharedPluginFiles/PluginUtils18_0.js

```diff
@@ -5,14 +5,34 @@ if (typeof FormIt === 'undefined')
 
 FormIt.PluginUtils = {};
 
-// Function to conver the given length into the current FormIt units.
+// convert the given length into the current FormIt units
```

Comment:
Add comment
// This function only works on the FormIt side

Reply:
Done.